neural_compressor/adaptor/onnxrt.py

```diff
@@ -1632,25 +1632,36 @@ def quantize(self, tune_cfg, model, data_loader, q_func=None):
         Returns:
             (dict): quantized model
         """
+        if self.performance_only:
+            tmp_model = model
+        else:
+            try:
+                tmp_model = copy.deepcopy(model)
```

Comment:
why need deep copy

Reply:
deepcopy the origin FP32 model for tuning